### ISSUE TITLE
Site does not provide any signatures

### ISSUE THREAD

**Issue by genodeftest** [rust-lang/rust-www#112]
The website http://www.rust-lang.org/ does not provide any signatures for downloads. This is a security threat.

**Comment by jimmycuadra** [comment]
Agreed – I was looking for this information and can't find it anywhere.

In addition, it looks like the site is being hosted on GitHub Pages, which means HTTP URLs are used and the HTTPS version of the URL uses GitHub's certificate, which is invalid for the rust-lang.org domain and gives a warning. Users who realize what's going on can accept the invalid certificate, but this is very poor UX at best, and for the majority who will not even attempt to use HTTPS, any signatures provided over HTTP can't really be trusted anyway.

**Comment by steveklabnik** [comment]
@jimmycuadra there's already two or three issues for that aspect.

@genodeftest we do have https://static.rust-lang.org/dist/staging/index.html but surfacing it would be nice.

(this page is a bit of a mess because a script went a little wrong, @edunham is working on taking down the .asc.asc.asc files...)

**Comment by steveklabnik** [comment]
https://static.rust-lang.org/dist/ too
